universal-login-wallet/src/integration/ethereum/TokensDetailsStore.ts

```diff
@@ -0,0 +1,21 @@
+import {TokenDetails, TokenDetailsService} from '@universal-login/commons';
+
+class TokensDetailsStore {
+  tokensDetails: TokenDetails[] = [];
+
+  constructor(private tokenDetailsService: TokenDetailsService, private tokensAddresses: string[]) {}
+
+  async aggregateTokensDetails() {
```

Comment:
`aggregateTokensDetails`  -> `fetchTokensDetails`

Aggregate relates to operations that produce scalar from vector/matrix e.g. sum, avg, dev